dockernet/tests/integration_tests.bats

```diff
@@ -152,6 +152,34 @@ setup_file() {
   assert_equal "$diff" $STAKE_AMOUNT
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] packet forwarding automatically liquid stake and ibc transfer stAsset to original network" {
```

Comment:
might be a bit tough to add this to the current test framework since we often run tests with only one zone

I think we should definitely try to test this manually though!

Reply:
Good point! Added a todo above to check this manually

Reply:
I tested this manually the other week, but would be good to have someone else go through it as well!

Reply:
tested this offline as a group. resolving